app/views/dashboard/_discovery_widget_host.html.erb

```diff
@@ -0,0 +1,4 @@
+<td><%= discovery_status_icon(host) %>&nbsp;<%= link_to trunc_with_tooltip(h(host.name)), discovered_host_path(host) %></td>
```

Comment:
Was wondering if we can do a smaller blank space. It looks like `&nbsp;` is maybe too much. Maybe to add just few pixels to the `span` that holds the icon. Smaller the better.